app/views/landing_page/blocks/_main_navigation.html.erb

```diff
@@ -3,7 +3,7 @@
 %>
 
 <div class="app-b-main-nav" data-module="app-b-main-navigation">
-  <p class="app-b-main-nav__heading govuk-heading-s"><%= block.data["title"]%></p>
+  <p class="app-b-main-nav__heading-p"><a class="govuk-heading-s govuk-link govuk-link--no-underline app-b-main-nav__heading-link" href="<%= block.data["title_link"]%>"><%= block.data["title"]%></a></p>
```

Comment:
Maybe use `link_to` here?

```
<%= link_to block.data["title"], block.data["title_link"], class: "govuk-heading-s govuk-link govuk-link--no-underline app-b-main-nav__heading-link" %>
```

Reply:
@andysellick Good point - done, thanks.  I've also moved `govuk-heading-s` back onto the paragraph, as I just noticed that having it on the link meant the active link's border was a few pixels above the block below it, whereas our designs have the border sitting nicely on top of the block below it.
<img width="623" alt="image" src="https://github.com/user-attachments/assets/15bc236a-52e5-4dbf-939e-78805d653345">